.github/workflows/build.yml

```diff
@@ -1,18 +1,14 @@
 name: Build
-
-on: [push, pull_request]
-
-env:
-  SCARB_VERSION: 0.7.0
-
+on:
+  push:
+  pull_request:
 jobs:
-  build:
+  check:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
-      - name: Set up Scarb
-        uses: software-mansion/setup-scarb@v1
-      - name: Check cairo format
-        run: scarb fmt --check
-      - name: Build cairo programs
-        run: scarb build
\ No newline at end of file
+      - uses: software-mansion/setup-scarb@v1
+        with:
+          scarb-version: "0.7.0"
+      - run: scarb fmt --check
+      - run: scarb test
```

Comment:
why run scarb test in the `build` ? when you also run scarb test in test scrypt ?

Reply:
typo, thanks